packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,46 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-none), var(--surface-shadow-none);
+  background-color: var(--surface-color-background-white);
+  max-width: var(--walkthrough-card-max-width);
+  flex-direction: column;
+  font-size: 14px;
+  color: var(--support-border-color-info);
+  padding: var(--layout-padding-large);
+  margin: var(--layout-margin-medium);
+  gap: var(--layout-margin-medium);
+  border: var(--layout-border-width-none);
+  text-align: left;
+  transition: box-shadow var(--transition-medium);
+}
+
+.ui-WalkthroughCard--Title {
+  color: var(--support-border-color-info);
+  font: var(--typography-bold-h5);
+  text-decoration: var(--walkthrough-card-title-text-decoration-hidden);
+  transition: text-decoration-color var(--transition-medium);
+  text-underline-offset: var(--walkthrough-card-title-text-underline-offset);
+}
+
+.ui-WalkthroughCard--CardContainer:hover {
```

Comment:
Can we use [data-hovered] to match the paradigms that react aria uses? Take a look at how button.css/link.css are handling it. We might want to also consider something for the [data-focus-visible] state as well.

Reply:
Since the card isn't a specific aria component like the button or link then no we can't use that...

Reply:
Ahh, previously it was using the Button component so we could have.